WordPress/src/main/java/org/wordpress/android/ui/people/PeopleAdapter.java

```diff
@@ -8,45 +8,50 @@
 import android.widget.TextView;
 
 import org.wordpress.android.R;
+import org.wordpress.android.datasets.PeopleTable;
 import org.wordpress.android.models.Person;
 import org.wordpress.android.models.Role;
 import org.wordpress.android.util.GravatarUtils;
 import org.wordpress.android.widgets.WPNetworkImageView;
 
-import java.util.ArrayList;
 import java.util.List;
 
 public class PeopleAdapter extends BaseAdapter {
     private final Context mContext;
     private final LayoutInflater mInflater;
-    private List<Person> mPersonList;
+    private List<Person> mPeopleList;
     private int mAvatarSz;
+    private int mLocalBlogId;
```

Comment:
Keeping a local state variable with the local blogId is an indication of coupling the adapter to the underlying database representation, which is probably not needed. See my comment below about `notifyDataSetChanged` for an alternative. Cheers!